src/components/currency/Currency.js

```diff
@@ -54,8 +49,8 @@ export class CurrencyComponent extends NumberComponent {
       mask: createNumberMask({
         prefix: this.prefix,
         suffix: this.suffix,
-        thousandsSeparatorSymbol: _.get(this.component, 'thousandsSeparator', this.delimiter),
-        decimalSymbol: _.get(this.component, 'decimalSymbol', this.decimalSeparator),
+        thousandsSeparatorSymbol: _.get(this.component, 'thousandsSeparator', this.overrideLanguage.delimiter || this.delimiter),
```

Comment:
This doesn't seem right. Instead of creating an override language and then checking it here, we should set scope.thousandsSeparator and scope.decimalSeparator when they are added to scope.